front-end/src/index.js

```diff
@@ -6,9 +6,9 @@ import './styles/index.css'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  //<React.StrictMode>
```

Comment:
Why remove strict mode here? I think its best to keep it at least for now. Its meant to help make sure the components it wraps follows certain best practices and its good for helping to detect bugs

Reply:
Will add back. Thank you